test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_smppy.py

```diff
@@ -42,7 +42,7 @@
 
 def _skip_if_image_is_not_compatible_with_smppy(image_uri):
     _, framework_version = get_framework_and_version_from_tag(image_uri)
-    compatible_versions = SpecifierSet("==2.11.*")
+    compatible_versions = SpecifierSet("==2.11.*,==2.12.*")
```

Comment:
This doesn't work. Please change this to
```suggestion
    compatible_versions = SpecifierSet(">=2.11.*,<2.13")
```